meta-spec-0000.md

```diff
@@ -70,11 +70,11 @@ Accepting a SPEC means that is is assigned a number,
 included in the [SPEC listing]({{< relref "/spec_overview.md" >}}),
 and linked to a
 [`specs` discussion topic](https://discuss.scientific-python.org/c/specs/accepted/15).
-It typically is accepted once there is some agreement that the SPEC
+The main criterion for acceptance is some agreement among the SSC that the SPEC
```

Comment:
The crux of this suggestion is to emphasize that it is the role of the SSC (not projects, individuals, or core projects) to determine whether to accept a SPEC.